ddtrace/constants.py

```diff
@@ -10,6 +10,7 @@
 _SINGLE_SPAN_SAMPLING_MAX_PER_SEC_NO_LIMIT = -1
 
 ORIGIN_KEY = "_dd.origin"
+_SPAN_LINKS_KEY = "_dd.span_links"
```

Comment:
-> `ddtrace/internal/constants.py`?

Reply:
This caused weird cython import errors:  https://github.com/DataDog/dd-trace-py/blob/bd77e58ded4eac06331b8b77505bf73423d028fe/ddtrace/internal/_encoding.pyx#L12. `ddtrace.internal.contants` shadows `ddtrace.constants`. I'll find a work around.

Reply:
🤔

Reply:
I moved the constant. The cython errros were unrelated. I was doing something dumb